url.js

```diff
@@ -76,9 +76,12 @@
     var instance = URLUtils(url || '');
 
     // Detect for ES5 getter/setter support
-    var ES5_GET_SET = (Object.defineProperties && (function () {
-      var o = {}; Object.defineProperties(o, { p: { 'get': function () { return true; } } }); return o.p;
-    }()));
+    var ES5_GET_SET = false;
+    try {
+      ES5_GET_SET = (Object.defineProperties && (function () {
+        var o = {}; Object.defineProperties(o, { p: { 'get': function () { return true; } } }); return o.p;
+      }()));
+    } catch (e) {};
```

Comment:
Can you use _ for an ignored variable? (Borrowing from Python style...)

No need for a trailing ; after the catch {} block